### ISSUE TITLE
Error: either set letsencrypt_account_email or add ''--register-unsafely-without-email'' to letsencrypt_opts_extra'

### ISSUE THREAD

**Issue by patsevanton** [systemli/ansible-role-letsencrypt#55]
Hello! Thanks for role letsencrypt
Try install 
### Playbook
```
- hosts: letsencrypt
  become: true
  roles:
    - role: systemli.letsencrypt
```

Inventory
```
all:
  children:
    letsencrypt:
      hosts:
        "letsencrypt":
          ansible_host: "xxx"
  vars:
    letsencrypt_cert:
      name: letsencrypt.xxxx.sslip.io
      domains:
        - letsencrypt.xxxx.sslip.io
      challenge: http
      http_auth: webroot
      opts_extra: "--register-unsafely-without-email"
      webroot_path: /var/www/letsencrypt.xxxxx.sslip.io
      services:
        - apache2
```

Output
```
TASK [systemli.letsencrypt : check if letsencrypt_account_email is set] ****************************************************************************************************************
Tuesday 26 April 2022  10:50:00 +0600 (0:00:01.529)       0:00:48.926 *********
fatal: [letsencrypt]: FAILED! => changed=false
  msg: 'Error: either set letsencrypt_account_email or add ''--register-unsafely-without-email'' to letsencrypt_opts_extra'
```

**Comment by patsevanton** [comment]
Fix https://github.com/systemli/ansible-role-letsencrypt/pull/56

**Comment by xshadow** [comment]
Fixed in #56
